### PR TITLE
Make storage pumper batch size configurable

### PR DESCRIPTION
**Pull request by Napoleon0621**
Add storage pumper batch size to config file
#185

### CODE REVIEW

**Comment by git-hulk** [comment]
I think we should put this config into `SecondaryStorage` instead of each redis config.

**Comment by Napoleon0621** [comment]
Please have another look, thx.

File: storage/manager.go [hunk 1]
```diff
@@ -130,11 +134,11 @@ func (m *Manager) PumpFn(name string, pool engine.Engine, threshold int64) func(
 			return false
 		}
 		metrics.storageDelJobs.WithLabelValues(name).Add(float64(len(jobsID)))
-		return len(jobsID) == maxJobBatchSize
+		return int64(len(jobsID)) == batchSize
 	}
 }
 
-func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64) {
+func (m *Manager) AddPool(name string, pool engine.Engine, threshold int64, maxPumpBatchSize int64) {
```
Comment (git-hulk) [inline storage/manager.go:141]: Remove this parameter. The manager can store the config when creating, then get this batch size from the config.

Comment (Napoleon0621) [reply storage/manager.go]: Just updated, please take another look.

**Review by git-hulk — APPROVED** [review]

**Pull request merged**